code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -157,7 +157,7 @@
 
 /obj/item/weapon/reagent_containers/pill/paracetamol
 	name = "Paracetamol pill"
-	desc = "Tylenol! A painkiller for the ages. Chewables!"
+	desc = "Tylenol! A painkiller for the ages. Chewables! Contains 15 units of Paracetamol."

```

Comment:
How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated. Also, TYLENOL is always capitalized.

Reply:
Just copied it from what it was before, the existence of Tylenol is completely coincidental and in no way related to TYLENOL®

Reply:
Tylenol is just a brand name for the actual painkiller that's become synonymous with the painkiller itself in the US, like how band-aid is just a brand name for a bandage

Reply:
TYLENOL is just a brand name for paracetamol, isn't it? And we don't use 'band-aid' ingame so far as I can tell

Reply:
Yeah, p sure it's paracetamol, and that was more an irl example

Reply:
Just call it paracetamol, that's what it is everywhere else in the code.